src/workerd/api/streams/internal.c++

```diff
@@ -572,8 +577,8 @@ kj::Maybe<jsg::Promise<ReadResult>> ReadableStreamInternalController::read(
       }
       readPending = true;
 
-      // TODO(now): This is the implementation we should be able to move to except there's a bug
-      // somewhere in the awaitIo implementation that allows IdentityTransformStream to hang.
```

Comment:
FWIW, this comment is just an old obsolete comment that should have been deleted before.